src/background/messaging/rpc-methods/send-transfer.ts

```diff
@@ -16,7 +20,12 @@ import {
   triggerRequestWindowOpen,
 } from '../messaging-utils';
 
-export async function rpcSendTransfer(message: SendTransferRequest, port: chrome.runtime.Port) {
+export async function rpcSendTransfer(
+  message: RpcRequest<'sendTransfer', RpcSendTransferParams | SendTransferRequestParams>,
+  port: chrome.runtime.Port
+) {
+  let params = message.params as RpcSendTransferParams;
```

Comment:
Imperative coding style. How can we do without reassigning var?

Reply:
refactored a bit
but leave variable, we need to assign somewhere converted params 🤔